src/App.css

```diff
@@ -244,7 +237,37 @@ input {
 
 .BackButton_List:hover {
   transform: scale(1.2);
-  transition: .1s;
+  transition: 0.1s;
   transition-timing-function: ease-in;
+}
 
+.duplicateMessage {
+  position: fixed;
+  top: 15%;
+  left: 50%;
+  -webkit-transform: translate(-50%, -50%);
+  transform: translate(-50%, -50%);
```

Comment:
You won't need the `-webkit-` prefix since `transform` is widely supported already 💯 

Docs on `transform` https://developer.mozilla.org/en-US/docs/Web/CSS/transform